### ISSUE TITLE
First child border style overwritten

### ISSUE THREAD

**Issue by Fraham** [rhyneav/papercss#6]
When the first child border has custom styles, for example border colour, this style is overwritten by the initial border styling
```css
.border, .border-1, .child-borders > *:nth-child(6n+1) {
    border: 2px solid @primary;
    border-top-left-radius: 255px 15px;
    border-top-right-radius: 15px 225px;
    border-bottom-right-radius: 225px 15px;
    border-bottom-left-radius: 15px 255px;
}
```

**Comment by Fraham** [comment]
#8 Fixes this issue

**Comment by rhyneav** [comment]
Ah nice catch! Just merged in #8 for you.

**Comment by rhyneav** [comment]
Went back and created a separate .border class with only the border size and color. This way it shouldn't override the child border as you mentioned. Without this, borders wouldn't have a size and would then be invisible.
```
.border {
    border: 2px solid @primary;
}
```

Does this work for you?
